assets/css/base/_base.scss

```diff
@@ -1,7 +1,10 @@
+@import 'typography';
```

Comment:
We import this in `css/base.scss`, I think you were adding this here because otherwise the SASS variables you are defining in the typography partial would not be accessible? 

Two ways to mitigate this, both cleaning up the structure a bit:
- Define the SASS variables in `abstracts/variables` (or in `base/_fonts` where the font weights and families are defined)
- Change the SASS variables to custom properties and define them in `base/custom-properties`